$OpenBSD: patch-src_squidclamav_c,v 1.3 2016/01/11 17:53:05 sthen Exp $

All the other buffers rx'ing data from clamav in this file use LBUFSIZ (32K).
Suspect this use of BUFSIZ from stdio.h (1K) is a typo. Trips the "backwards
memcpy" check but obviously isn't.

--- src/squidclamav.c.orig	Sat Oct 17 10:35:42 2015
+++ src/squidclamav.c	Mon Jan 11 17:46:03 2016
@@ -1854,7 +1854,7 @@ int squidclamav_safebrowsing(ci_request_t * req, char 
     debugs(1, "DEBUG Ok connected to clamd socket.\n");
 
     debugs(1, "DEBUG: Scanning url for Malware now\n");
-    uint32_t buf[BUFSIZ/sizeof(uint32_t)];
+    uint32_t buf[LBUFSIZ/sizeof(uint32_t)];
     strcpy(cbuff, "From test\n\n<a href=");
     strncat(cbuff, url, MAX_URL_SIZE);
     strcat(cbuff, ">squidclamav-safebrowsing-test</a>\n");
